express-api/package.json

```diff
@@ -41,7 +41,7 @@
     "typeorm": "0.3.20",
     "typeorm-naming-strategies": "4.1.0",
     "winston": "3.13.0",
-    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz",
+    "xlsx": "0.18.5",
```

Comment:
This package is a weird outlier. The version on NPM (0.18.5) is outdated and has some vulnerabilities.
The site (https://docs.sheetjs.com/docs/getting-started/installation/nodejs) shows they want it installed from the tgz instead.